### ISSUE TITLE
Socket Plugin documentation

### ISSUE THREAD

**Issue by ltotsas** [ngxs/store#907]
<pre><code>
[ ] Regression 
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[x] Documentation issue or request
[ ] Support request
[x] Other... Please describe: 
</code></pre>

## Expected behavior
It would be very beneficial if the socket plugin had better documentation, or even better an example showing the most important functionality. I think that this goes for all the plugins as examples are missing for every plugin.

## Problem
This is the error that i get from console
```
ConnectWebSocket Object { action: {…}, status: "DISPATCHED" } socket.service.ts:16:38
ConnectWebSocket  Object { action: {…}, status: "SUCCESSFUL" } socket.service.ts:16:38
ConnectWebSocket Object { payload: undefined } socket.service.ts:19:27
{…} . devices.component.ts:18:66
devices: {…}
devices: Array []
<prototype>: Object { … }
<prototype>: Object { … }

ERROR Error: "Type undefined not found on message"
WebSocketHandler ngxs-websocket-plugin.js:309
```

## Stackblitz mini demo
https://stackblitz.com/edit/angular-buglzg

## Environment

<pre><code>
Libs:
- @angular/core version: 7.0.0
- @ngxs/store version: 3.4.2

</code></pre>

**Comment by markwhitfeld** [comment]
@ltotsas We would welcome any contribution to the docs that you would like to make.
Always looking for help with improving the docs 😉

**Comment by splincode** [comment]
Released in 3.5.0
